src/helpers/storage.ts

```diff
@@ -1,31 +1,38 @@
 import fs from 'fs';
 import path from 'path';
+import url from 'url';
 import { CONFIG_FILE_NAME, ERR_LOG_FILE, FIRMWARE_FILE_NAME, INITIAL_APP_STATE, STD_LOG_FILE } from './constants.js';
 import { State } from './types';
 
+const dirname = Boolean(import.meta.url) ? url.fileURLToPath(new URL('.', import.meta.url)) : __dirname;
```

Comment:
Did you test this in node 10? it could fail on import being undefined

Reply:
yes. tested it in all 4 ways of running this project